src/core/eff-spec.ts

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+import * as mocha from 'mocha';
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+
+import {Op, Eff} from './eff';
+import {Run} from './run';
+import {EffUtil} from './effutil';
+
+chai.should();
```

Comment:
Would it be possible to use the `expect` interface of `chai`, which doesn't monkey-patch `Object.prototype`?